exercises/simple-cipher/.meta/template.j2

```diff
@@ -0,0 +1,57 @@
+{%- import "generator_macros.j2" as macros with context -%}
+import re
+{{ macros.header(imports=['Cipher']) }}
+
```

Comment:
To make this work, you'll need to apply the following changes to `bin/generate_tests.py`. This is a filter that should've been there from the beginning, and I have no problem adding it as part of this PR:
```
diff --git a/bin/generate_tests.py b/bin/generate_tests.py
index 3a602cf..61475c8 100755
--- a/bin/generate_tests.py
+++ b/bin/generate_tests.py
@@ -95,6 +95,10 @@ def has_error_case(cases):
     return False
 
 
+def regex_replace(s, find, repl):
+    return re.sub(find, repl, s)
+
+
 def load_canonical(exercise, spec_path):
     """
     Loads the canonical data for an exercise as a nested dictionary
@@ -206,6 +210,7 @@ def generate(
     env = Environment(loader=loader, keep_trailing_newline=True)
     env.filters['to_snake'] = to_snake
     env.filters['camel_case'] = camel_case
+    env.filters['regex_replace'] = regex_replace
     env.tests['error_case'] = error_case
     result = True
     for exercise in glob(os.path.join('exercises', exercise_glob)):
```

Suggested Change:
```suggestion
{%- macro convert_js(s) -%}
{{ s |
    regex_replace("\\.substring\\((\\d+), ([^)]+)\\)", "[\\1:\\2]") |
    regex_replace("(\w+)\\.length", "len(\\1)")
}}
{%- endmacro %}

```